### DISCUSSION TITLE
No in-app chat

### DISCUSSION THREAD

**Discussion by 0ttik** [codemagic-ci-cd/codemagic-docs#1087] [Q&A] [answered]
I have billing enabled as a team, but no in-app chat is visible, how do I get a support via in-app chat?

**Comment by mikhail-tokarev** [comment, marked as answer]
note that some Ad Blockers might block the chat widget

**Reply by 0ttik** [reply]
Oh, right, thank you! Disabling ad-blocker restored in-app chat icon in the right corner of the codemagic site.
